doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -33,8 +32,15 @@ ifdef::DM[]
 In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 +
-** `rhdm-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} replicated pods that build a service from source. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and do not need to use any other tools to manage the pods.
 endif::DM[]
+** `{PRODUCT_INIT}-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build services from source or pull them from a Maven repository. You can then replicate each pod as necessary. 
++
+You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, deploy a new server image to replace the old one. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
++
+When configuring this environment, you must customize the {KIE_SERVER} in the configuration console and click the *Set immutable server configuration* button. 
```

Comment:
configuration console does not fits well here.;

Reply:
We don't use the term "wizard".